wp-content/themes/great-outdoor/header.php

```diff
@@ -39,7 +39,9 @@
 
         <header class="header">
 
-            <a href="<?php echo esc_url( bloginfo('url') ); ?>"><h1><?php esc_html( bloginfo('name') ); ?></h1></a>
+            <a href="<?php echo esc_url( bloginfo('url') ); ?>">
+                <img src="<?php echo get_stylesheet_directory_uri() . '/assets/logo.png' ?>" alt="GOPC logo">
```

Comment:
To keep things explicit, I just added an id to the image. It probably doesn't need to be nested like `header #masthead` but I went with it. Thanks for the feedback to help things move along more smoothly.